### PR TITLE
Adding flows that assess the correctness of the major types of automations

### PR DESCRIPTION
**Pull request by chrisguidry**
For each of these cases, we (re)create an automation that is looking for very
specific conditions that are unique to the test (using a unique resource ID).
We then emit the events necessary to make the automation fire, then wait for it
to be triggered (or timeout and fail if it doesn't happen in time).

Note: I wasn't sure how to make an effective integration test for metric
triggers at this time, given the longer time horizons.  This is something we
can work on over time.

### CODE REVIEW

**Comment by chrisguidry** [comment]
Oops thank you!

File: flows/ensure-integration-automations.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        response.raise_for_status()
+
+        automation = response.json()
+        logger.info("Created automation %s", automation["id"])
+
+        return automation
+
+
+@flow
+async def ensure_integrations_automations():
```
Comment (chrisguidry) [inline flows/ensure-integration-automations.py:28]: @jamiezieziula Here's where we'll re-add the automations that I _think_ we'd want in this workspace

**Review by jamiezieziula — COMMENTED** [review]
sorry last thing - you'll need to add the flow names to the github workflow!

**Comment by chrisguidry** [comment]
Ohhh gotcha, on it

**Review by jamiezieziula — APPROVED** [review]

**Pull request merged**